src/main/java/spoon/support/compiler/jdt/CompilationUnitWrapper.java

```diff
@@ -43,30 +44,14 @@
 			return jdtCompiler.loadedContent.get(s);
```

Comment:
No, it does not.
The cache is cleared before each compilation (see JDTBatchCompiler constructor)